Easydict/Swift/Feature/Configuration/Configuration.swift

```diff
@@ -588,6 +624,26 @@ extension Configuration {
     fileprivate func didSetAppearance(_ appearance: AppearenceType) {
         DarkModeManager.sharedManager().updateDarkMode(appearance.rawValue)
     }
+
+    fileprivate func didSetAppleDictShortcut() {
+        let windowManager = EZWindowManager.shared()
+        windowManager.updateWindowsTitlebar()
+    }
+
+    fileprivate func didSetPinShortcut() {
+        let windowManager = EZWindowManager.shared()
+        windowManager.updateWindowsTitlebar()
+    }
+
+    fileprivate func didSetGoogleShortcut() {
+        let windowManager = EZWindowManager.shared()
+        windowManager.updateWindowsTitlebar()
+    }
+
+    fileprivate func didSetEudicDictShortcut() {
+        let windowManager = EZWindowManager.shared()
+        windowManager.updateWindowsTitlebar()
+    }
```

Comment:
It looks like they are using the same logic, it would be better to replace them with a general function.

Reply:
Actually, I mean we could remove the `didSetAppleDictShortcut ` and other 3 functions and use  `updateWindowTitlebar ` when receive Defaults value changes. What do you think?